realTimeChat/server/helpers/index.js

```diff
@@ -0,0 +1,40 @@
+const usersList = [];
+
+const getIndividualRoomUsers = (room) => {
+  return usersList.filter((user) => user.room === room);
+};
+
+const AddNewUser = (user) => {
+  usersList.push(user);
+  return usersList;
+};
+
+const removeUserFromRoom = (socketId) => {
+  const index = usersList.findIndex((user) => user.id === socketId);
+
+  if (index !== -1) {
+    return usersList.splice(index, 1)[0];
+  }
+};
```

Comment:
This would become inefficient with large number of rooms and users. Using a room-user map instead could speed up the performance